test/fixtures/members/purgeCache.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Purge Cache Response
+ * Multiple responses can be added to the array if required
+ *
+ * @return {Object}
+ */
+
+module.exports = () => {
+  return [
+    {
+      message: "Cache purged successfully",
+      success: true,
+      errors: [],
+      messages: [],
+      result: {
+        id: "ba637cab83d148e6935cbba0b1abcxyz",
+      },
+    },
```

Comment:
For Discord Discussion - https://discord.com/channels/673083527624916993/1000471526174359663